sdk/go/common/resource/config/map_test.go

```diff
@@ -1225,15 +1252,13 @@ func TestSetFail(t *testing.T) {
 		{Key: "my:root["},
 		{Key: `my:root["nested]`},
 		{Key: "my:root.array[abc]"},
-		{Key: "my:root.[1]"},
```

Comment:
Why is this no longer a syntax error?

Reply:
IIRC the property path parser is a bit loose in its interpretation of `.`, and treats empty property names as no-ops.